src/services.js

```diff
@@ -122,6 +122,26 @@ const tilesetStatus = function (tilesetId) {
   setTimeout(checkStatus, 10000, tilesetId);
 };
 
+// Check job status and report warnings
+const jobStatus = async function (tilesetId, jobId) {
```

Comment:
For these functions, can you wrap them in a try/catch like the other functions in this file?